src/instructlab/model/chat.py

```diff
@@ -789,6 +536,157 @@ def start_prompt(
         self._update_conversation(response_content.plain, "assistant")
 
 
+def chat_model(
+    ctx,
```

Comment:
ctx shouldn't be in here, when this is an API call it won't be able to pickle the context obj, please pass in the required fields from the config needed